R/fuse_df.R

```diff
@@ -0,0 +1,89 @@
+
+#' Fuse Degrees-of-Freedom Columns into Variable Labels
+#'
+#' Takes the output from [apa_print()] methods and modifies the results table
+#' by fusing information about degrees of freedom into the variable labels of
+#' test-statistic columns.
+#'
+#' @param x    Either the complete output object created by [apa_print()] methods,
+#'   or the `table` component of such objects.
+#' @param quietly Logical. If FALSE (the default), checks if df columns contain non-integer values.
+#' @param ... further arguments passed from an to other methods
+#'
+#' @examples
+#'   apa_out <- apa_print(aov(yield ~ N * P, npk))
+#'
+#'   # Standard output with separate columns for degrees of freedom:
+#'   apa_out
```

Comment:
To highlight what happens here, should we adapt the example to use `apa_out$table` and `fuse_df(apa_out)$table`?